Source/Plugins/Core/com.equella.core/js/tsrc/modules/SearchModule.ts

```diff
@@ -247,18 +247,23 @@ export const generateCategoryWhereQuery = (
 export const queryStringParamsToSearchOptions = async (
   location: Location
 ): Promise<SearchOptions | undefined> => {
-  if (!location.search) return undefined;
-  const params = new URLSearchParams(location.search);
-
-  // If no query strings is of type LegacySearchParams, return undefined.
-  if (!Array.from(params.keys()).some((key) => LegacySearchParams.guard(key))) {
+  if (!location.search) {
     return undefined;
   }
-
-  if (location.pathname.endsWith("searching.do")) {
+  const params = new URLSearchParams(location.search);
+  const searchOptions = params.get("searchOptions");
```

Comment:
Seems rather verbose, would it be fine to have:

```suggestion
  const searchOptions = new URLSearchParams(location.search).get("searchOptions");
```

Reply:
Both `params`  and  'searchOptions' are needed in the below `if` block.

Reply:
Aaaah yes, of course. I should've read ahead before commenting. ;)